### PR TITLE
feat: iOS additional onboarding step

### PR DESCRIPTION
**Pull request by AmarTrebinjac**
## Changes
Implements the PWA install screen for iOS on Safari.

## Events

Did you introduce any new tracking events?

## Experiment

`onboarding_pwa`

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://mi-629.preview.app.daily.dev

### CODE REVIEW

File: packages/webapp/pages/onboarding.tsx [hunk 1]
```diff
@@ -13,7 +13,10 @@ import AuthOptions, {
   AuthProps,
 } from '@dailydotdev/shared/src/components/auth/AuthOptions';
 import { AuthTriggers } from '@dailydotdev/shared/src/lib/auth';
-import { OnboardingHeader } from '@dailydotdev/shared/src/components/onboarding';
+import {
+  OnboardingHeader,
+  OnboardingPWA,
```
Comment (rebelchris) [inline packages/webapp/pages/onboarding.tsx:18]: Please lazy load this one as it includes a video and gets very little use :)

**Review by rebelchris — CHANGES_REQUESTED** [review]
Nice work, some very minor changes needed.

File: packages/shared/src/components/onboarding/OnboardingPWA.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React, { type ReactElement } from 'react';
+import { OnboardingTitle } from './common';
+import { cloudinaryPWA, cloudinaryPWAVideo } from '../../lib/image';
+import { Typography } from '../typography/Typography';
+
+export const OnboardingPWA = (): ReactElement => {
+  return (
+    <>
```
Comment (rebelchris) [inline packages/shared/src/components/onboarding/OnboardingPWA.tsx:8]: Extra fragment not needed

**Review by rebelchris — APPROVED** [review]
You didn't change the one isPWA, but nothing breaking :D
Do check prod well for iOS and non iOS users

**Comment by AmarTrebinjac** [comment]
> You didn't change the one isPWA, but nothing breaking :D Do check prod well for iOS and non iOS users

@rebelchris I had to run out for an errand, so stopped in the middle of it ! 😅 

Will implement the CTA change + abstraction. Will also remove the footer again, as discussed [recently here](https://dailydotdev.slack.com/archives/C0381BQUEN9/p1733811070377239?thread_ts=1732780166.207429&cid=C0381BQUEN9)

Will also update the position of the button on the android onboarding and also abstract there while I'm at it :)

**Pull request merged**